docs/build/security-docs/threat-modeling/STRIDE-template.mdx

```diff
@@ -0,0 +1,68 @@
+---
+sidebar_position: 30
+sidebar_label: STRIDE Threat Model Template
+title: STRIDE Threat Model Template
```

Comment:
if both `sidebar_label` and `title` are intended to be the same, you don't need to include the `sidebar_label`. it'll just default to the title